src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""BKSF transform on fermionic operators."""
+#from __future__ import absolute_import
+
+import numpy as np
+from fermilib.ops import InteractionOperator
+from projectq.ops import QubitOperator
+import networkx as nx
```

Comment:
How badly do you need this package? We really try to avoid unusual package dependancies when we can and for good reason. Often, these obscure external packages are not maintained to the engineering standards that we strive for. Perhaps this one is. But it is something extra that must be installed and we do try to avoid that. If this provides a data structure that we can write ourselves with little effort, we should. Or, if there is a more common package (for instance, one that comes with the basic anaconda python), that that would be preferred. If it provides something truly unique and also truly complex and difficult to implement ourselves, then we need it and it is okay. But justification is required for adding new packages to our dependancies.